createwindow/window_linux.go

```diff
@@ -25,3 +26,25 @@ func ChromeCommand(layout Layout, userdatadir, display string, debuggingPort int
 	}
 	return cmd
 }
+
+func DisplayError(err error, cmdErrorHandler func(err error) error, isFatal bool) error {
+	messageType := "warning"
+	if isFatal {
+		messageType = "error"
+	}
+
+	zenity := command.ExternalCommand{
+		Path: "zenity",
+		Arg: []string{
+			fmt.Sprintf("--%s", messageType),
+			fmt.Sprintf("--title=\"%s message\"", messageType),
```

Comment:
Okay, this is consistent now, but to prevent escaping double quotes you should use raw strings.

```
fmt.Sprintf(`--title="%s message"`, messageType)
```

same below.

Also, I don't understand why %s message is in double quotes of its own, but the html string below isn't (did you try this? I think the title of your prompt will _actually_ have double quotes in this case).
Maybe the older way (without "=") was easier to reason about (then neither would have double quotes).

My only ask is that you be consistent, i.e. not use "=" syntax in width either.